app/components/display-page.tsx

```diff
@@ -3,9 +3,10 @@ import { Center, Flex, Image, Heading } from '@chakra-ui/react';
 interface DisplayPageProps {
   img: string;
   desc: string;
+  alt: string;
 }
 
-export default function DisplayPage({ img, desc }: DisplayPageProps) {
+export default function DisplayPage({ img, desc, alt }: DisplayPageProps) {
```

Comment:
While you are touching this file, can you please rename it?  "DisplayPage" doesn't tell you anything about what this is for.